tests/test_admin.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+from django.core.files.base import File
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+import pytest
+from pyquery import PyQuery as pq
+from zds_client.client import ClientError
+
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+from zgw_consumers.test import mock_service_oas_get
+
+API_ROOT = "https://example.com/api/v1/"
+OAS_PATH = Path(__file__).parent / "schemas/ztc.yaml"
+TESTS_DIR = Path(__file__).parent
+
+
+@pytest.mark.django_db
+def test_listzaaktypen_mixin(settings, admin_client, requests_mock):
+    """
+    Assert that an error message is displayed in the admin if ClientError is raised
+    """
+    settings.ZGW_CONSUMERS_TEST_SCHEMA_DIRS = [TESTS_DIR / "schemas"]
+    mock_service_oas_get(requests_mock, API_ROOT, "ztc")
+
+    # define errors
+    error_cause = ClientError()
+    error_cause.args = [{"status": 400, "detail": "some other error"}]
+    err = ClientError()
+    err.args = [
+        {
+            "title": "Je hebt geen toestemming om deze actie uit te voeren.",
+            "status": 403,
+            "detail": "Client identifier bestaat niet",
+        }
+    ]
+    err.__cause__ = error_cause
+
+    requests_mock.get(f"{API_ROOT}zaaktypen", exc=err)
+
+    # set up service
+    with open(OAS_PATH, "r") as oas_file:
+        service = Service.objects.create(
+            label="Test",
+            api_type=APITypes.ztc,
+            api_root=API_ROOT,
+            oas_file=File(oas_file, name="schema.yaml"),
+        )
+        service.full_clean()
+
+    # assert that admin page works despite ClientError
+    url = reverse("admin:testapp_zgwconfig_change")
+    response = admin_client.get(url)
+    assert response.status_code == 200
+
+    html = response.content.decode("utf-8")
+    doc = pq(html)
+
+    # assert that error message is displayed
+    error = doc(".error")[0]
+    assert error.text == _(
+        "Failed to retrieve available zaaktypen (got 403 - Client identifier bestaat niet). "
+        "The cause of this exception was: 400 - some other error"
+    )
+
+    # assert that Zaaktype field is present in admin page despite ClientError
+    zaaktypen_label = doc.find("label")
+    assert zaaktypen_label.text() == _("Zaaktype:")
```

Comment:
I'm certain that the `:` is not part of the translatable message - especially because your model does not specify an explicit `verbose_name`. So Django is just taking the model field name, replacing underscores with spaces and capitalizes the first character.

```suggestion
    assert zaaktypen_label.text() == "Zaaktype:"
```